packages/invoices/src/OutboundShipment/DetailView/ContentArea.tsx

```diff
@@ -8,16 +8,17 @@ import {
   MiniTable,
   useIsGrouped,
 } from '@openmsupply-client/common';
-import { InvoiceLine, InvoiceItem } from '../../types';
+import { OutboundItem } from '../../types';
```

Comment:
essentially just replacing the `InvoiceItem` with `OutboundShipmentLineFragment` (Mayeb that should be `OutboundLineFragment` 🤔 

This is so we don't have to manage the `InvoiceLine` type which is a pain. An example of annoyance is that because it extends `InvoiceLineNode`, the `item` field is an `ItemNode`.. so that means it has the required fields `stats` and `availableBatches`, when all you really want is a more simple subset

Reply:
For some reason, didn't see the comments while reviewing. Might have opened the PR too early!
Agree with the rename of `OutboundShipmentLineFragment` to `OutboundLineFragment` 😁